source/aggregation-pipeline-builder.txt

```diff
@@ -165,11 +193,56 @@ letter case and accent marks.
 
 To specify a custom collation:
 
-#. Click the :guilabel:`Collation` button at the top of the pipeline
-   builder.
+#. Click :guilabel:`More Options` in the top right corner of the pipeline 
+   builder. 
+
+   .. figure:: /images/compass/options-collation.png
+        :alt: Collation Option
 
 #. Enter your collation document.
 
+.. _export-agg-builder-results:
+
+Export Aggregation Results
+--------------------------
+
+To export results from your aggregation pipeline:
+
+#. Connect to the deployment containing the
+   :doc:`collection </collections>` you wish to export data from.
+
+   To learn how to connect to a deployment, see
+   :ref:`connect-run-compass`.
+
+#. Navigate to your desired collection.
```

Comment:
Minor, but I think that "target" is a more appropriate word than "desired".

```suggestion
#. Navigate to your target collection.
```

Reply:
Done.